src/utils/notification.js

```diff
@@ -0,0 +1,55 @@
+// Publishes messages to SNS and consumes them from SQS.
+'use strict';
+const config = require('../config');
+const logger = require('../logger').getLogger(module);
+const SNS = require('sns.js');
+const SQS = require('sqs-consumer');
+
+const eventType = {
+  APARTMENT_CREATED: 'APARTMENT_CREATED',
+  APARTMENT_LISTED: 'APARTMENT_LISTED',
+  APARTMENT_UNLISTED: 'APARTMENT_UNLISTED',
+  APARTMENT_RENTED: 'APARTMENT_RENTED'
+};
+
+/* Publish a single message to AWS SNS topic.
+ * Work example:  yield notifications.publish(notifications.eventType.APARTMENT_CREATED, 
+ *                  { apartment_id: 1, user_uuid: '211312-4123123-5344-234234-2343' });
+ */
+function* publish(eventType, dataPayload) {
+  let message = {
+    eventType,
```

Comment:
@avnersorek, Is there anything you can think of, besides event type, that should not be part of data payload?